shard.yml

```diff
@@ -1,8 +1,6 @@
 name: ameba
 version: 0.13.2
```

Comment:
Should the version be bumped up?

Reply:
Not in this PR, no.

Reply:
@drhuffman12 do you need a new tagged release? I can issue one.

Reply:
@veelenga Yes please.  https://github.com/athena-framework/validator/pull/9/checks?check_run_id=1496168419

Reply:
@Blacksmoke16 @drhuffman12 published https://github.com/crystal-ameba/ameba/releases/tag/v0.13.3